eng/source-build-patches/0001-Dont-build-analyzers-for-source-build.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/eng/Analyzers.props b/eng/Analyzers.props
+index 438ae20..cd2b97c 100644
+--- a/eng/Analyzers.props
++++ b/eng/Analyzers.props
+@@ -1,9 +1,9 @@
+ <Project>
+-  <PropertyGroup>
++  <PropertyGroup Condition="'$(ArcadeBuildFromSource)' != 'true'">
```

Comment:
If you want to check this in, I'd be OK with it.

Reply:
I think the flag that has historically been used is `DotNetBuildFromSource`:

https://github.com/dotnet/source-build/blob/master/Documentation/RepoApi.md#parameters

Reply:
@hoyosjs - I checked this in and eliminated the patch.

@omajid - Updated to use `DotNetBuildFromSource`